docs/src/modules/components/Demo.js

```diff
@@ -74,7 +75,9 @@ class Demo extends React.Component<any, any> {
           <CodeIcon />
         </IconButton>
         <Collapse in={this.state.codeOpen}>
-          <MarkdownElement className={classes.code} text={`\`\`\`js\n${raw}\n\`\`\``} />
+          <NoSSR>
+            <MarkdownElement className={classes.code} text={`\`\`\`js\n${raw}\n\`\`\``} />
```

Comment:
Speed up the time to first paint. This part isn't visible and needs the page to be interactive to be shown.